demo/containers/App.jsx

```diff
@@ -66,7 +66,8 @@ class App extends React.Component {
         </header>
 
         <section>
-          <h2>Demo</h2>
+          <h2>Demo: <CountUp start={3} end={0} duration={10} redraw /></h2>
+          <p><i>This 3,2,1 count-down has an optional redraw prop, making its animation re-run at every render.</i></p>
```

Comment:
I would prefer to keep the demo as simple as it is. But we could add this at the [advance section](https://github.com/glennreyes/react-countup#advanced) in the [`README.md`](https://github.com/glennreyes/react-countup/blob/master/README.md).
Also we need to add doc to this prop. 👍